pbconf/Vagrantfile

```diff
@@ -23,7 +23,8 @@ Vagrant.configure("2") do |config|
   end
 
   config.vm.synced_folder "..", "/home/vagrant/prj/fossology", type: "rsync"
-  config.vm.network "forwarded_port", guest: 80, host: 8081
+  config.vm.synced_folder "../target", "/home/vagrant/repo", type: "rsync", create: true
```

Comment:
It might be better if that is not of type rsync. At least for me the rsync does not automatically work.